configure

```diff
@@ -1173,11 +1175,22 @@ case "${ARCH}" in
 
         # Enable arch-specific optimizations?
         if test $without_optimizations -eq 0; then
-            CFLAGS="${CFLAGS} -DX86_CPUID -DX86_SSE2 -DX86_SSE2_CHUNKSET -DX86_SSE42_CRC_HASH"
-            SFLAGS="${SFLAGS} -DX86_CPUID -DX86_SSE2 -DX86_SSE2_CHUNKSET -DX86_SSE42_CRC_HASH"
+            CFLAGS="${CFLAGS} -DX86_CPUID"
+            SFLAGS="${SFLAGS} -DX86_CPUID"
+            ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} x86.o"
+            ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} x86.lo"
 
-            ARCH_STATIC_OBJS="${ARCH_STATIC_OBJS} x86.o chunkset_sse.o insert_string_sse.o slide_sse.o"
-            ARCH_SHARED_OBJS="${ARCH_SHARED_OBJS} x86.lo chunkset_sse.lo insert_string_sse.lo slide_sse.lo"
+            # if(WITH_SSE4 AND (HAVE_SSE42CRC_INLINE_ASM OR HAVE_SSE42CRC_INTRIN))
+            CFLAGS="${CFLAGS} -DX86_SSE42_CRC_HASH"
```

Comment:
Assuming SSE4.2 is available is still bad as a lot of AMD processors don't have SSE4.2 as it was originally available on only Intel processors. I don't know when the test was removed as even my previous development machine didn't have SSE4.2 and zlib-ng worked on it using configure...

Reply:
I think there was never a check in configure; I pulled in the check from CMakeLists.txt and
added it as a comment in configure.  I admit that's a bit confusing.  I suspect zlib-ng checks
for sse4.2 at runtime before using it...?